File: Basic Problem Solving/count_digits/countDigit.js
const countDigit = num => {
  return Math.floor(Math.log10(num) + 1)
}

const bruteForceCountDigit = num => {
  let count = 0
  while (num > 0) {
    num = Math.floor(num / 10)
    count++
  }
  return count
}

console.log(countDigit(4356), bruteForceCountDigit(4356))

File: Basic Problem Solving/count_digits/countDigit.py
import math
def countDigit(num) :
    return int(math.log10(num) + 1)

def bruteForcecountDigit(num) :
    cnt = 0
    while num > 0:
        num = num//10
        cnt += 1
    return cnt

ans = countDigit(4368)
ans2 = bruteForcecountDigit(4368)
print(ans,ans2)